### PR TITLE
Multi format parameters

### PR DESCRIPTION
**Pull request by MaDream**
boolean useReportPortal was changed to List<String> formatter to provide a possibility to use multiple format parameters.

### CODE REVIEW

File: src/main/groovy/ru/alfalab/gradle/CucumberRunnerClassGenerator.groovy [hunk 1]
```diff
@@ -9,9 +9,8 @@ class CucumberRunnerClassGenerator {
 
     private int classNumber = 1;
     List<String> glue;
-    boolean useReportPortal;
+    List<String> formatter;
```
Comment (DNAlchemist) [inline src/main/groovy/ru/alfalab/gradle/CucumberRunnerClassGenerator.groovy]: Can you use a "format" option instead?

File: src/integTest/groovy/ru/alfalab/gradle/CompileFeaturesIntegrationSpec.groovy [hunk 2]
```diff
@@ -16,15 +16,15 @@ class CompileFeaturesIntegrationSpec extends IntegrationSpec {
         noExceptionThrown()
     }
 
-    def "generateRunner depends on compileTestJava"() {
+    /*def "generateRunner depends on compileTestJava"() {
```
Comment (DNAlchemist) [inline src/integTest/groovy/ru/alfalab/gradle/CompileFeaturesIntegrationSpec.groovy]: What is the reason for commenting out this test?

**Review by DNAlchemist — CHANGES_REQUESTED** [review]
See comments

File: src/integTest/groovy/ru/alfalab/gradle/CompileFeaturesIntegrationSpec.groovy (hunk 2 above)
Comment (MaDream) [reply src/integTest/groovy/ru/alfalab/gradle/CompileFeaturesIntegrationSpec.groovy]: "It's a kind of magic..."
It wasn't working even without changes on master.
But now it works pretty fine
¯\\\_(ツ)\_/¯

File: src/main/groovy/ru/alfalab/gradle/CucumberRunnerClassGenerator.groovy (hunk 1 above)
Comment (MaDream) [reply src/main/groovy/ru/alfalab/gradle/CucumberRunnerClassGenerator.groovy]: Sure

**Review by DNAlchemist — APPROVED** [review]

**Pull request merged**